docker/client/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:buster AS toolchain
```

Comment:
This seems to be a duplicate of https://github.com/libra/libra/blob/master/docker/client/client.Dockerfile  Similarly for mint and validator-dynamic's.  We can git mv each one.  

Or maybe I am missing the intent?

Reply:
NM, found the other shoe in https://github.com/libra/libra/pull/3279